### PR TITLE
bundle support for jailed devices

### PR DESCRIPTION
**Pull request by dayanch96**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,7 @@
+ifeq ($(ROOTLESS),1)
```
Comment (PoomSmart) [inline Makefile:1]: This isn't needed for sideload support PR.

Comment (dayanch96) [reply Makefile:1]: its to simplify make package for rootless. I'll remove it if its not needed as well. Going to add localization support in few mins

Comment (PoomSmart) [reply Makefile:1]: You make a fair point, then again it should be in a separate PR.

Comment (dayanch96) [reply Makefile:1]: Got you. Should i separate PRs also for localization support and small UI improvements?

**Comment by dayanch96** [comment]
separated. creating new pr for localization support will conflict with this branch. will wait for this one

File: Makefile (hunk 1 above)
Comment (PoomSmart) [reply Makefile]: Ideally, yeah. Separate.

**Pull request merged**